src/config.json

```diff
@@ -2,6 +2,10 @@
   "BACKEND_URL": false,
   "DATA_SOURCES": [
     {
+      "tracks": [{"files": [{"name": "snp1kg-BRCA1.vg.xg", "type": "graph"}]},
+                 {"files": [{"name": null, "type": "haplotype"}]},
+                 {"files": [{"name": "NA12878-BRCA1.sorted.gam", "type": "read"}]}],
+                   
       "name": "snp1kg-BRCA1",
       "graphFile": "snp1kg-BRCA1.vg.xg",
```

Comment:
The original `xxxFile` fields (except `bedFile`) aren't used anymore, right? We should probably drop them.